### PR TITLE
Now, JPG images can be uploaded using batch loader

### PR DESCRIPTION
**Pull request by BeeramVineethReddy**
## Summary

In the previous version of the software, the user is not allowed to upload the images of jpg, jpeg, or png format.

But in many cases, the images to be visualized might be in the jpg format. It is a painful job to convert all the jpg image data to pyramidal tiff format, then upload and visualize. It would be easy for the user if we allow uploading of jpg images also.

## Motivation

the issue(regarding the uploading of JPG images) is reported as issue number: #458

### CODE REVIEW

File: apps/batchloader/batchLoader.js [hunk 1]
```diff
@@ -293,7 +295,7 @@ function updateFileName(oldfileName) {
     let newFileName = $('#newFileName');
     let newName = newFileName.val();
     let fileExtension = newName.toLowerCase().split('.').reverse()[0];
-
+    alert(fileExtension);
```
Comment (birm) [inline apps/batchloader/batchLoader.js:298]: Did you indend this just for dev purposes?

Comment (BeeramVineethReddy) [reply apps/batchloader/batchLoader.js:298]: Yes, I used it(line 298) just for testing purposes..!

Comment (BeeramVineethReddy) [reply apps/batchloader/batchLoader.js:298]: We can just comment it out if we don't want to see an alter popup in the browser while uploading.

Comment (birm) [reply apps/batchloader/batchLoader.js:298]:
```suggestion
```

**Comment by akhil-rana** [comment]
Haven't been in regular touch with the development recently. So are JPEGs now supported by the server too?

File: apps/batchloader/batchLoader.js (hunk 1 above)
Comment (BeeramVineethReddy) [reply apps/batchloader/batchLoader.js]: The code is modified as per the suggestions made .. with the commit message "Suggested Changes Made" ..!

**Review by birm — CHANGES_REQUESTED** [review]
So, we've added some support for a quick conversion of jpg/png images primarily so that developers/new users don't need to find WSI images. However, this was added to loader.js in this commit - https://github.com/camicroscope/caMicroscope/commit/0554794ea063c5390da3fbd81223aa83e439bd41

However, this has not been carried over to the batch loader. For some reason I was under the impression that they were using the same code, but that seems not to be the case. As a result, this doesn't seem to work now.

For next steps, I'd see this as either reducing code duplication by using methods from loader.js for both loaders, or duplicating the functionality into batchLoader.js.

**Pull request closed without merging**